lib/info-extras.js

```diff
@@ -268,3 +268,54 @@ exports.cleanVideoDetails = videoDetails => {
     'videoDetails.shortDescription', 'videoDetails.description');
   return videoDetails;
 };
+
+/**
+ * Get storyboards info.
+ *
+ * @param {Object} info
+ * @returns {Array.<Object>}
+ */
+exports.getStoryboards = info => {
+  const parts = info.player_response.storyboards &&
+    info.player_response.storyboards.playerStoryboardSpecRenderer &&
+    info.player_response.storyboards.playerStoryboardSpecRenderer.spec &&
+    info.player_response.storyboards.playerStoryboardSpecRenderer.spec.split('|');
+
+  if (!parts) return [];
+
+  const url = new URL(parts.shift());
+
+  return parts.map((part, i) => {
+    let [
+      width,
+      height,
+      count,
+      storyboardWidth,
+      storyboardHeight,
+      interval,
```

Comment:
i'm seeing interval be `0` for some, is that expected?

Reply:
Yes, this is the default storyboard, the lowest quality and the small thumbnails represent bigger time spans, this should be used if the others are not available or if the network is slow to show something instread of blank